webapp/channels/src/components/advanced_text_editor/send_button/send_post_options/core_menu_options.tsx

```diff
@@ -67,14 +105,25 @@ function CoreMenuOptions({handleOnSelect}: Props) {
                     values={{'9amTime': timeComponent}}
                 />
             }
+            className='core-menu-options'
+            {...extraProps}
         />
     );
 
-    const nextMonday = moment().
-        tz(userTimezone).
-        day(8). // next monday; 1 = Monday, 8 = next Monday
-        set({hour: 9, minute: 0, second: 0, millisecond: 0}). // 9 AM
-        valueOf();
+    function getNextWeekday(dateTime: DateTime, targetWeekday: number) {
+        // eslint-disable-next-line no-mixed-operators
```

Comment:
This can be addressed in a separate PR since you're making more changes to this already, but we shouldn't be ignoring ESLint rules if we can avoid it

Reply:
got it